src/web/mjs/connectors/ComicBrise.mjs

```diff
@@ -0,0 +1,48 @@
+import SpeedBinb from './templates/SpeedBinb.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ComicBrise extends SpeedBinb {
+
+    constructor() {
+        super();
+        super.id = 'comicbrise';
+        super.label = 'comicBrise';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://comic-brise.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.post-title');
+        let id = uri.pathname;
+        let title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const msg = 'This website does not support mangas/chapters, please copy and paste the links containing the chapters directly from your browser into HakuNeko.';
```

Comment:
It seems website provide titles, why not support them?
https://comic-brise.com/titlelist/